yarn.lock

```diff
@@ -6794,6 +6806,19 @@ brfs@^2.0.2:
     static-module "^3.0.2"
     through2 "^2.0.0"
 
+broadcast-channel@^2.3.2:
```

Comment:
This module is pretty well-maintained, and it didn't look like any of the dependencies were alarming or ones that we'd otherwise want to avoid.